simple.css

```diff
@@ -53,7 +57,7 @@ progress {
   appearance: none;
   -webkit-appearance: none;
   -moz-appearance: none;
-}
+}   
```

Comment:
There's a stray whitespace change here, should be reverted :)

Reply:
I don't see this in upstream after cherry-picking the commits, resolving.